eclipse-theia/theia
# Fix rending of quickpick buttons

#### What it does

Ensure that  the Theia specific wrapper for the `MonacoQuickPickItem` properly forwards assignments of the "buttons" property to the wrapped item.

More context:
The buttons property is assigned in the [QuickOpenMainImpl](https://github.com/eclipse-theia/theia/blob/b15b639fa3b64a9518efdf3e82ea9098915ba145/packages/plugin-ext/src/main/browser/quick-open-main.ts#L272). This works perfectly fine when creating a `InputBox` via the quickopen service but for the normal `QuickPick`
this assignment gets lost because the actual monaco quickpick item is wrapped and the buttons property assignment is not forwarded.

Fixes #13076

Contributed on behalf of STMicroelectronics

#### How to test


Follow the steps to reproduced described in https://github.com/eclipse-theia/theia/issues/13076#issuecomment-1896065081
(vsix for testing is provided).
After installling the plugins., execute the `Hello World` command. Notice that buttons are now properly rendered (while not being rendered at all on master)
#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
